pytools/__init__.py

```diff
@@ -2994,6 +2999,15 @@ def strtobool(val: Optional[str], default: Optional[bool] = None) -> bool:
 # }}}
 
 
+# {{{ unique
+
+def unique(seq: Sequence[T]) -> Iterator[T]:
+    """Yield unique elements in *seq*. See also :func:`reversed`."""
+    yield from list(dict.fromkeys(seq))
```

Comment:
```suggestion
    return iter(dict.fromkeys())
```

Reply:
Saw your simplification: `yield from` is not zero cost, `return` is.

Reply:
> Saw your simplification: `yield from` is not zero cost, `return` is.

Thanks, `return iter` is indeed much faster. Done in [d4f9ab3](https://github.com/inducer/pytools/pull/187/commits/d4f9ab3135ddde4ccc1810041b9770865fdb6ade)